pajbot/modules/pnsl.py

```diff
@@ -55,6 +62,11 @@ def __init__(self, bot):
                 self.pnsl_token = bot.config["pnsl"].get("token", None)
 
     def run_pnsl(self, bot, source, message, event, args):
+        if self.settings["offline_only"] is True:
+            if not self.bot.is_online:
+                bot.whisper(source, f"{bot.streamer} is online! Skipping PNSL list eval.")
+                return False
```

Comment:
```suggestion
        if self.settings["offline_only"] and self.bot.is_online:
            bot.whisper(source, f"{bot.streamer_display} is live! Skipping PNSL list eval.")
            return False
```

First, `not self.bot.is_online` would have exactly the opposite effect (disallowing the run when offline), second you can collapse the nested if-blocks into one. Also, we should use the streamer's display name.